Front-page/style.css

```diff
@@ -0,0 +1,164 @@
+@import url("https://fonts.googleapis.com/css2?family=Readex+Pro&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Pacifico&display=swap");
+* {
+  font-family: "Readex Pro", sans-serif;
+  transition: all ease 0.5s;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --primary: blueviolet;
+  --secondary: #0b1120;
+  --font-color: #637084;
+  --white:#fff;
```

Comment:
put color keyword for colors
use only one convention for colors - either HEX codes or color names
Suggestion - use HEX codes